tower_cli/resources/job_template.py

```diff
@@ -99,17 +99,12 @@ class Resource(models.SurveyResource):
         help_text='On write commands, perform extra POST to the '
                   'survey_spec endpoint.')
 
-    @resources.command
-    def create(self, fail_on_found=False, force_on_exists=False,
-               extra_vars=None, **kwargs):
-        """Create a job template."""
+    def write(self, *args, **kwargs):
```

Comment:
```The standard options were getting applied, but somehow were not functioning correctly.```

I think I figured it out: It's all in the extra `extra_vars=None`, if you try doing nothing, but simply deleting that part, you will find the issue solved too.